src/Patching/MyWorkflow1Initial.workflow.cs

```diff
@@ -0,0 +1,21 @@
+using Temporalio.Workflows;
+
+namespace TemporalioSamples.Patching;
+
+[Workflow("MyWorkflow")]
+public class MyWorkflow1Initial
+{
+    private string? result;
+
+    [WorkflowRun]
+    public async Task RunAsync()
+    {
+        result = await Workflow.ExecuteActivityAsync(() => Activities.PrePatchActivity(), new() { ScheduleToCloseTimeout = TimeSpan.FromMinutes(5) });
+    }
+
+    [WorkflowQuery]
+    public string? Result()
+    {
+        return result;
+    }
```

Comment:
```suggestion
    [WorkflowQuery]
    public string? Result { get; private set; }
```

Cleaner to use properties IMO (and can remove field)

Reply:
Done